test/runnable/cdcmp.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+src_file=${EXTRA_FILES}/${TEST_NAME}.d
+expect_file=${EXTRA_FILES}/${TEST_NAME}.out
+tmp_file=${RESULTS_TEST_DIR}/${TEST_NAME}.out
+obj_file=${RESULTS_TEST_DIR}/${TEST_NAME}.o
+
+if [ $OS == "linux" ] && [ $MODEL == "64" ]; then
+  $DMD -c -O ${src_file} -of${obj_file}
+  objdump --disassemble --disassembler-options=intel-mnemonic ${obj_file} | tail -n+3 > ${tmp_file}
```

Comment:
What happens if the format of objdump just changes slightly?

Reply:
`cp` the changed file over the old one.

Reply:
Sure, but then auto-tester or another CI will fail because the output mismatches.

Reply:
Check in the changed file.